Source/DafnyCore/Backends/Rust/Dafny-compiler-rust.dfy

```diff
@@ -1861,6 +1866,10 @@ module {:extern "DCOMP"} DafnyToRustCompiler {
       } else {
         assume {:axiom} forall m: ModuleItem <- mod.body.value :: m < mod;
         var optExtern: ExternAttribute := ExtractExternMod(mod);
+        var attributes := [];
+        if HasTestAttribute(mod.attributes) {
+          attributes := [R.RawAttribute("#[cfg(test)]")];
+        }
```

Comment:
I'd rather not do exactly this because it's not a complete solution. `{:test}` on modules doesn't have any meaning at the Dafny level as it does for methods. And it's likely to be misinterpreted as "This scope contains test methods" as it is in some testing frameworks, rather than "This code should only be compiled in test mode". I suspect users won't realize they have to also put `{:test}` on helper modules as well.

If you really want this I'd suggest `{:rust_test_cfg}` or something like that instead.

Reply:
I'll go with {:rust_test_cfg} then, it will be easier later to have Backend-specific typed options.

Reply:
I also updated the docs to mention the two attributes in the Rust-specific section.